primitives/src/validator.rs

```diff
@@ -104,3 +106,22 @@ pub enum MessageTypes {
     Heartbeat(Heartbeat),
     Accounting(Accounting),
 }
+
+pub mod fixtures {
+    use fake::faker::*;
+
+    use crate::BigNum;
+
+    use super::ValidatorDesc;
+
+    pub fn get_validator<V: AsRef<str>>(validator_id: V, fee: Option<BigNum>) -> ValidatorDesc {
+        let fee = fee.unwrap_or_else(|| BigNum::from(<Faker as Number>::between(1, 13)));
+        let url = format!(
+            "http://{}-validator-url.com/validator",
```

Comment:
The URL would lead to a lot of errors when the validator tick is running because it will try to post heartbeat messages to the sentry node specified in this URL.
We should stick to the URL in the JS stack implementation i.e. 
```
validators: [
				{ id: 'awesomeLeader', url: 'http://localhost:8005', fee: '100' },
				{ id: 'awesomeFollower', url: 'http://localhost:8006', fee: '100' },
			]
```